### PR TITLE
Portfolio: validate contact form

### PR DESCRIPTION
**Pull request by Abenezer-Tilahun**
# Portfolio: validate contact form

In this Milestone , some of the tasks We accomplished were:

- Added validation checks to make sure email address is in lower case before submitting the form
- Added validation message that appears when a validation check has failed
- Added shake animation to validation message on display

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -273,31 +273,23 @@ <h3 class="heading-tertiary">Contact me</h3>
...
+            <label for="message">Message:</label>
+            <textarea class="contact_message" id="message" placeholder="Write your message here" maxlength="500" required></textarea>
           </li>
-          <li>
-            <p class="show-msg">Please make sure the email is all lower case!</p>
-            <input id="submit" type="submit" value="Get in touch" class="button btn-contact"/>
+          <li class="form_li_submit">
+            <p class="validation_message"></p>
+            <input type="submit" class="submit_btn" value="Get In Touch" />
           </li>
```
Comment (rustam-tolipov) [inline index.html:291]: ## BEAUTIFUL DESIGN 👏

- [x] You did a great job so far. However, there is just one issue in your design and I am sure you can fix it 😉. Please kindly fix the width of the validation message which is changing the width of the form. You can see the screenshot below 📸, it's getting wider

**Not submitted** 
image

**After I click the button**
image

**Review by rustam-tolipov — APPROVED** [review]
Hi @Abenezer-Tilahun , 

Good job so far!

### HIGHLIGHTS!

- No linters errors.💯
- Descriptive Readme.md. 💯


There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Meltrust — APPROVED** [review]
Hi @Abenezer-Tilahun, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (Abenezer-Tilahun) [reply index.html:291]: Done thanks for suggestion.

**Pull request merged**